### PR TITLE
Merge "timh"

### PR DESCRIPTION
**Pull request by jneidel**
Search function

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -14,6 +14,14 @@
                 </a>
             </div>
 
+            <div id="top-search">
+                <img id="top-search-icon" src="img/search.png" />
+            </div>
+            
+            <div id="top-search-box">
+                <input type="text" id="top-search-input" placeholder="Search Google..."></input>
+            </div>
```
Comment (jneidel) [inline app/index.html]: The id is showing that something is a child of another div.
For example top-search-box is child of top-search, that means that top-search-box as to be within the div of top-search 😄

Comment (jneidel) [inline app/index.html]: The id is showing that something is a child of another div.
For example top-search-box is child of top-search, that means that top-search-box as to be within the div of top-search 😄

File: app/script.js [hunk 2]
```diff
@@ -285,6 +285,17 @@ let quote = {
     }
 }
 
+var searchButton = document.getElementById("top-search-icon");
+var searchBox = document.getElementById("top-search-box");
```
Comment (jneidel) [inline app/script.js]: You can declare multiple variables with a comma after each one.

Example:
`var searchButton = document.getElementById("top-search-icon"),`
`searchBox = document.getElementById("top-search-box");`

File: app/script.js [hunk 3]
```diff
@@ -285,6 +285,17 @@ let quote = {
...
+
+searchButton.addEventListener("click", function() {
+    if (searchBox.style.display == "block") {
+        searchBox.style.display = "none";
+    } else {
+        searchBox.style.display = "block";
+    }
+})
```
Comment (jneidel) [inline app/script.js]: Missing a semicolon 😉

File: app/style.css [hunk 4]
```diff
@@ -15,7 +15,7 @@ body {
 }
 #top {
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-start;
```
Comment (jneidel) [inline app/style.css]: Please revert this back to space-between and wrap a div around the top left stuff instead.
(We'll need it for the weather later)

Comment (timOTdev) [reply app/style.css]: Okay. Didn't know about wrapping a div. If I change this, do I make a new commit?

Comment (jneidel) [reply app/style.css]: Yeah just do all the changes and then commit it. Will be included in this pull request

File: app/index.html (hunk 1 above)
Comment (timOTdev) [reply app/index.html]: If I change it to:
`<div id="top-input">
                <input type="text" id="top-input-box" placeholder="Search Google..."></input>
            </div>`

Would that work?

File: app/script.js (hunk 3 above)
Comment (timOTdev) [reply app/script.js]: Line 297?

Comment (jneidel) [reply app/script.js]: Yeah :smile:

File: app/index.html (hunk 1 above)
Comment (jneidel) [reply app/index.html]: Dont rename the div, its a part of the search box so put it inside the top-search div :smile:

**Pull request merged**